apps/tenant-management-webapp-e2e/src/integration/events/events.step.ts

```diff
@@ -175,3 +175,257 @@ Then('the user views the details of {string} under Core streams', function (stre
     .invoke('text')
     .should('contain', '"name": "' + streamName + '"');
 });
+
+Given('a tenant admin user is on event streams page', function () {
+  commonlib.tenantAdminDirectURLLogin(
+    Cypress.config().baseUrl,
+    Cypress.env('realm'),
+    Cypress.env('email'),
+    Cypress.env('password')
+  );
+  commonObj.adminMenuItem('menu-event').click();
+  commonObj.serviceTab('Event', 'Streams').click();
+  cy.wait(2000);
+});
+
+When('the user clicks Add stream button', function () {
+  eventsObj.addStreamBtn().click();
+});
+
+Then('the user views Add stream modal', function () {
+  eventsObj.streamModalTitle().invoke('text').should('eq', 'Add stream');
+});
+
+When(
```

Comment:
Since we have EMPTY and n/a at different places now. We should start to add a comment on top of the step definition bout n/a and which field allows n/a. Please add a comment there.